.gitmodules

```diff
@@ -1,3 +1,12 @@
-[submodule "dependencies/OpenCOLLADA"]
-	path = dependencies/OpenCOLLADA
-	url = https://github.com/KhronosGroup/OpenCOLLADA.git
\ No newline at end of file
+[submodule "dependencies/OpenCOLLADA/OpenCOLLADA"]
```

Comment:
Moving this down a level makes it very hard to switch back and forth between 1.0 and 2.0.  Maybe the 1.0 one can also be moved down a level, to match?